app/src/main/java/com/xmartlabs/template/helper/GeneralErrorHelper.java

```diff
@@ -67,32 +67,21 @@ public GeneralErrorHelper() {
     BaseProjectApplication.getContext().inject(this);
   }
 
-  private void logCrashlyticsError(ServiceExceptionWithMessage exceptionWithMessage) {
+  private void logError(ServiceExceptionWithMessage exceptionWithMessage) {
     String url = ServiceHelper.getUrl(exceptionWithMessage.getResponse().raw());
     int resultCode = exceptionWithMessage.getCode();
     String headers = exceptionWithMessage.getResponse().headers().toString();
     String body = exceptionWithMessage.getErrorBody();
-    Crashlytics.setString(CRASHLYTICS_KEY_URL, url);
```

Comment:
The information is sent in the message of the exception. What would the key and sending the information twice accomplish here?